cmd/server/config.go

```diff
@@ -89,6 +91,19 @@ func (c *config) Validate() error {
 		}
 	}
 
+	if len(c.BridgeTokenList) != 0 {
+		for _, token := range c.BridgeTokenList {
```

Comment:
Should we verify that the token address is a bridge token?

Reply:
There's not really a good check for what is/isn't a bridge token

Reply:
I wonder if it makes sense to hardcode the bridge tokens in rosetta cc @patrick-ogrady